lib/config/configItems.joi.js

```diff
@@ -51,10 +51,22 @@ const zenkoAuthJoi = joi.object({
     account: joi.string().required(),
 });
 
+const retryParamsJoi = joi.object({
```

Comment:
Seems like these default values are a combination of scality and public cloud destinations. Also, wondering why `0.2` jitter, whereas elsewhere it is consistently `0.1`.

Reply:
It's a remnant of changes that did not propagate. I will amend this, `0.2` or `0.1` has medium impact, so just sticking to one for consistency.